WebRoot/js/ajax/dwt/widgets/DwtListView.js

```diff
@@ -1794,7 +1794,7 @@ function(dropAllowed) {
 
 DwtListView.prototype._setNoResultsHtml =
 function() {
-	var	div = document.createElement("div");
+	var	div = this._getNoResultsContainer();
```

Comment:
minor: not added in this commit, but better to replace tab with space between var and div words